src/dlgMapper.cpp

```diff
@@ -197,26 +197,22 @@ void dlgMapper::updateAreaComboBox()
     comboBox_showArea->setCurrentText(oldValue); // Try and reset to previous value
 }
 
-void dlgMapper::slot_toggleShowRoomIDs(int s)
+void dlgMapper::slot_toggleShowRoomIDs(int toggle)
 {
-    if (s == Qt::Checked) {
-        mp2dMap->mShowRoomID = true;
-    } else {
-        mp2dMap->mShowRoomID = false;
-    }
+    mp2dMap->mShowRoomID = toggle == Qt::Checked;
     mp2dMap->mpHost->mShowRoomID = mp2dMap->mShowRoomID;
     mp2dMap->update();
 }
 
-void dlgMapper::slot_toggleShowRoomNames(int s)
+void dlgMapper::slot_toggleShowRoomNames(int toggle)
 {
-    mpMap->setRoomNamesShown(s == Qt::Checked);
+    mpMap->setRoomNamesShown(toggle == Qt::Checked);
     mp2dMap->update();
 }
 
-void dlgMapper::slot_toggleStrongHighlight(int v)
+void dlgMapper::slot_toggleStrongHighlight(int toggle)
 {
-    mpHost->mMapStrongHighlight = v == Qt::Checked ? true : false;
+    mpHost->mMapStrongHighlight = toggle == Qt::Checked ? true : false;
```

Comment:
As above, for clarity, and besides the ternary operator is redundant as the comparison already produces a `bool`result:
```suggestion
    mpHost->mMapStrongHighlight = (toggle == Qt::Checked);
```
BTW This method hasn't been used for, I think, 10 years, as it fell out of the UI a long-time ago (somewhere between Mudlet 2.1 and 3.0 IIRC) maybe we want to restore it!

Reply:
Your action may be needed.

Reply:
Applied.